src/test/fakeExtensionContext.ts

```diff
@@ -23,6 +23,7 @@ export class FakeExtensionContext implements ExtensionContext {
     public storagePath: string | undefined
     public globalStoragePath: string = '.'
     public logPath: string = ''
+    public extensionUri: Uri = Uri.parse('')
```

Comment:
where is this used ?

Reply:
I'm honestly not sure. Removed it and the linter isn't yelling at me so I guess I'll roll with it.